docs/APIs/storage/block_device.md

```diff
@@ -28,6 +28,14 @@ Block devices indicate their block sizes through the `get_read_size`, `get_progr
 
 As a rule of thumb, you can use the erase size for applications that use a single block size (for example, the FAT file system).
 
+## Utility block devices
+
+mbed OS contains several utility block devices to give you better control over how storage is allocated.
+
+- The [slicing block device](https://github.com/ARMmbed/mbed-os/blob/master/features/filesystem/bd/SlicingBlockDevice.h) allows you to partition storage into smaller block devices that you can use independentally.
+- The [chaining block device](https://github.com/ARMmbed/mbed-os/blob/master/features/filesystem/bd/ChainingBlockDevice.h) allows you to chain multiple block devices together and extend the usable amount of storage.
+- The [MBR block device](https://github.com/ARMmbed/mbed-os/blob/master/features/filesystem/bd/MBRBlockDevice.h) allows you to partition storage with a [Master Boot Record](https://en.wikipedia.org/wiki/Master_boot_record). The MBR allows the partitions to be configured seperately from outside the application.
```

Comment:
Hmm, so:

> With the [Master Boot Record (MBR) block device](https://github.com/ARMmbed/mbed-os/blob/master/features/filesystem/bd/MBRBlockDevice.h), you can partition storage with a [master boot record](https://en.wikipedia.org/wiki/Cattle#/media/File:CH_cow_2_cropped.jpg). The MBR allows the partitions to be configured seperately from outside the application.

Seems a bit weird since the acronym is a part of the name. 

How about something like this?

> mbed OS comes with support for storing partitions on disk with a [Master Boot Record (MBR)](https://en.wikipedia.org/wiki/Cattle#/media/File:CH_cow_2_cropped.jpg). The [MBR block device](https://github.com/ARMmbed/mbed-os/blob/master/features/filesystem/bd/MBRBlockDevice.h) provides this functionality and supports creating partitions at runtime or using preformatted partitions configured seperately from outside the application.

Also I'm not really aware of a better source than wikipedia for the MBR. The MBR came about as an ad hoc portability layer between competing companies, so there never was a governing standard published by any authority. The MBR design has been mostly reverse engineered and I think wikipedia has the best collection of info. I'm open for a different source, but I don't think we can beat the wikipedia article in terms of being up to date.

There is the os-dev wiki entry on MBRs, but it's also a wiki (and is more focused on the linux point of view):
http://wiki.osdev.org/MBR_(x86)